packages/zosfiles/__tests__/__system__/api/methods/invoke/DefineVSAM.ams.temp

```diff
@@ -0,0 +1 @@
+DEFINE CLUSTER (NAME (UZUKO01.ZOSFILE.VSAM.A1550767.A897943) CYLINDERS (5 2 ) VOLUMES())
```

Comment:
Is this file committed by accident?

Reply:
yes - it was generated by tests, I think.